src/main/java/bookstore/controller/OrderController.java

```diff
@@ -0,0 +1,72 @@
+package bookstore.controller;
+
+import bookstore.dto.order.OrderDto;
+import bookstore.dto.order.OrderItemDto;
+import bookstore.model.Order;
+import bookstore.model.User;
+import bookstore.service.OrderService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.web.PageableDefault;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.security.core.Authentication;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "Order", description = "Manage order")
+@RestController
+@RequestMapping("/api/orders")
+@RequiredArgsConstructor
+public class OrderController {
+    private final OrderService orderService;
+
+    @PostMapping
+    @PreAuthorize("hasRole('ROLE_USER')")
+    @Operation(summary = "Place an order",
+            description = "Place an order and clear the shopping cart")
+    public OrderDto createOrder(Authentication authentication, @RequestBody String shippingAdress) {
```

Comment:
Create object for storing shipping address 
```suggestion
    public OrderDto createOrder(Authentication authentication, Object///) {
```